Fix PowerPC detection (needs __linux__) and add aarch64 support.

Patch sent upstream: https://freeswitch.org/jira/browse/FS-8746

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

diff -uNr freeswitch-1.6.6.org/libs/libzrtp/include/zrtp_config.h freeswitch-1.6.6/libs/libzrtp/include/zrtp_config.h
--- freeswitch-1.6.6.org/libs/libzrtp/include/zrtp_config.h	2016-01-12 17:32:36.000000000 +0100
+++ freeswitch-1.6.6/libs/libzrtp/include/zrtp_config.h	2016-01-16 14:09:17.830860603 +0100
@@ -21,7 +21,7 @@
 #		define ZRTP_PLATFORM ZP_ANDROID
 #	elif defined(__FreeBSD__)
 #		define ZRTP_PLATFORM ZP_BSD
-#	elif defined(linux) || defined(__linux)
+#	elif defined(linux) || defined(__linux__)
 #		include <linux/version.h>
 #		define ZRTP_PLATFORM ZP_LINUX
 #	elif defined(__MACOSX__) || defined (__APPLE__) || defined (__MACH__)
@@ -88,7 +88,13 @@
  */
 #define ZRTP_BYTE_ORDER ZBO_BIG_ENDIAN
 
-#elif defined(ARM) || defined(_ARM_) || defined(ARMV4) || defined(__arm__)
+#elif defined(__AARCH64EB__)
+/*
+ * aarch64, big endian
+ */
+#define ZRTP_BYTE_ORDER ZBO_BIG_ENDIAN
+
+#elif defined(ARM) || defined(_ARM_) || defined(ARMV4) || defined(__arm__) || defined(__AARCH64EL__)
 /*
  * ARM, default to little endian
  */
